interface/globals.php

```diff
@@ -462,6 +462,9 @@ function undoMagicQuotes($array, $topLevel=true) {
 $userauthorized = empty($_SESSION['userauthorized']) ? 0 : $_SESSION['userauthorized'];
 $groupname = empty($_SESSION['authProvider']) ? 0 : $_SESSION['authProvider'];
 
+$attendant_type = (empty($pid) && isset($_SESSION['therapy_group'])) ? 'gid' : 'pid';
```

Comment:
place some comments above these lines so developers don't step on these in the future.